_includes/CheatSheet.md

```diff
@@ -1,213 +1,214 @@
 
-## LocalDate
+Note that getter methods for instance properties omit the get keyword: `d.year()`, not ~~`d.getYear()`~~.
+
+### Try it out
+
+**Tip:** You can run the examples on this page in your browser console. All `js-joda` classes are imported into the global namespace of this web page.
 
-A LocalDate represents a date without a time and time-zone in the ISO-8601 calendar system, such as 2007-12-24.
+![image](https://user-images.githubusercontent.com/2136620/47266364-9be10b00-d535-11e8-9a38-b53a6c16728d.png)
 
-### Create a LocalDate
+## LocalDate
+
+A `LocalDate` represents a date with **no time** and **no time zone** in the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) calendar system, such as 2007-12-24.
 
-```javascript
+### Create a `LocalDate`
 
-// obtain the current date in the system default timezone, e.g. 2016-02-23
+```JavaScript
+// obtain the current date in the system default time zone, e.g. 2016-02-23
 LocalDate.now();
 
-// obtain the current date in the utc timezone, e.g. 2016-02-23
+// obtain the current date in the UTC time zone, e.g. 2016-02-23
 LocalDate.now(ZoneOffset.UTC);
 
-// obtain an instance of LocalDate from an ISO8601 formatted text string
-LocalDate.parse('2016-02-23');
+// obtain an instance of LocalDate from an ISO 8601 formatted text string
+LocalDate.parse("2016-02-23");
 
 // obtain an instance of LocalDate from a year, month, and dayOfMonth value
-LocalDate.of(2016, 2, 23) // 2016-02-23
+LocalDate.of(2016, 2, 23); // 2016-02-23
 
 // obtain an instance of LocalDate from a year, month, and dayOfMonth value
-LocalDate.of(2016, Month.FEBRUARY, 23) // 2016-02-23
-
-// obtain an instance of LocalDate from am epochDay where day 0 is 1970-01-01
-LocalDate.ofEpochDay(-1) // 1969-12-31
+LocalDate.of(2016, Month.FEBRUARY, 23); // 2016-02-23
 
-// obtain an instance of LocalDate from am epochDay where day 0 is 1970-01-01
-LocalDate.ofYearDay(2016, 42) // 2016-02-11
+// obtain an instance of LocalDate from an epochDay where day 0 is 1970-01-01
+LocalDate.ofEpochDay(-1); // 1969-12-31
 
+// obtain an instance of LocalDate from an epochDay where day 0 is 1970-01-01
+LocalDate.ofYearDay(2016, 42); // 2016-02-11
 ```
 
-### Get values from LocalDate
+### Get values from `LocalDate`
 
-```javascript
+```JavaScript
+const d = LocalDate.parse("2016-12-24");
 
-var d = LocalDate.parse('2016-12-24');
+d.toString(); // '2016-12-24' ISO 8601 format
 
-d.toString();   // '2016-12-24' ISO8601 format
+d.dayOfMonth(); //        24
+d.month(); //             Month.DECEMBER
+d.monthValue(); //        12
+d.year(); //              2016
 
-d.dayOfMonth(); // 24
-d.month();      // Month.DECEMBER
-d.monthValue(); // 12
-d.year();       // 2016
-
-d.dayOfWeek();         // DayOfWeek.SATURDAY
+d.dayOfWeek(); //         DayOfWeek.SATURDAY
 d.dayOfWeek().value(); // 6
-d.dayOfYear();         // 359
+d.dayOfYear(); //         359
```

Comment:
i'd prefer to either have the `//` aligned, or have the comment value directly behind the `//` separated with a single space ... but not align values and not align comment `//`

Reply:
again... please consistent throughout the file :), i won't change each occurence :)